recipe/meta.yaml

```diff
@@ -18,28 +17,39 @@ source:
     # into various problems around ABIs of numpy/C++ etc.
     # Hopefully, we can provide more specific builds per CPU architecture
     # in the future, see conda-forge/conda-forge.github.io#1261
-    - patches/0002-silence-log-that-tells-people-to-install-from-pip.patch
+    - patches/0001-silence-log-that-tells-people-to-install-from-pip.patch
     # backport nmslib/nmslib#488 for compatibility with newer pybind11
-    - patches/0003-Fix-import-on-newer-versions-of-pybind11-fixes-471.patch
+    # https://github.com/nmslib/nmslib/pull/488
+    - patches/0002-Fix-import-on-newer-versions-of-pybind11-fixes-471.patch
+    # Remove compiler flag that causes build failure
+    - patches/0003-remove-march-compiler-flag.patch
+    # Fix pragma message
+    - patches/0004-fix-pragma-message.patch
+    - patches/0005-remove-installation-requirements-handled-by-conda.patch
     # some weird interaction with pybind11 + quote escapes;
-    # upstream CI need quotes escaped, we need them unescaped
-    - patches/0004-do-not-escape-quotes-for-VERSION_INFO.patch
+    # upstream CI need quotes escaped, we (seems to be an issue on win64 only) need them unescaped
+    - patches/0006-do-not-escape-quotes-for-VERSION_INFO.patch  # [win]
 
 build:
-  number: 2
-  script: {{ PYTHON }} -m pip install ./python_bindings -vv
+  number: 0
+  script: {{ PYTHON }} -m pip install ./python_bindings --no-deps --no-build-isolation -vv
+  missing_dso_whitelist:  # [s390x]
+    - '$RPATH/ld64.so.1'  # [s390x]
```

Comment:
I don't think $RPATH is necessary, since it is going to be searched by default.

Reply:
To be honest I just copied what had been done for other packages, to be consistent. Most of them use $RPATH/ld64.so.1:

aggregate find ./* -type f -name 'meta.yaml' -exec grep -H -E 'ld64.so.1' {} \;

./abseil-cpp-feedstock/recipe/meta.yaml:    - '$RPATH/ld64.so.1'  # [s390x]
./boost-1.73-feedstock/recipe/meta.yaml:        - '$RPATH/ld64.so.1'  # [s390x]
./capnproto-feedstock/recipe/meta.yaml:    - $RPATH/ld64.so.1    # [linux]
./cargo-bundle-licenses-feedstock/recipe/meta.yaml:    - $RPATH/ld64.so.1  # [s390x]
./cffi-feedstock/recipe/meta.yaml:    - $RPATH/ld64.so.1  # [s390x]
./cryptography-feedstock/recipe/meta.yaml:    - $RPATH/ld64.so.1    # [s390x]
./cython-blis-feedstock/recipe/meta.yaml:    - $RPATH/ld64.so.1    # [linux and s390x]
./gdbm-feedstock/recipe/meta.yaml:    - $RPATH/ld64.so.1    # [linux and s390x]

Reply:
yes, I saw the same in other feedstock, I think that is OK

Reply:
Can you try without $RPATH and see if it builds? I think if RPATH is searched by default, even though other recipes may include it, it should not be included (and we should set the right example ;) )

Reply:
It fails:
https://cloud.prefect.io/anaconda/flow-run/c4fd1791-85f0-4e14-b594-12ad59bd7251?logs

We should use either '$RPATH/ld64.so.1' or '*/ld64.so.1'

Reply:
RPATH has to be here always because the feedstock build is free to set up RPATH in any location